source/usage-examples/bulkWrite.txt

```diff
@@ -0,0 +1,86 @@
+=======================
+Perform Bulk Operations
+=======================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+MongoDB provides clients the ability to perform write operations in bulk.
+Bulk operations affect a *single* collection.
+
+The following code sample performs a bulk write operation against the
+``users`` collection in the ``sample_mflix`` database. Specifically, the
+program uses the :node-api:`collection.bulkWrite
+<Collection.html#bulkWrite>` method and ``insertOne`` operations to write
+the user information from a ``users.json`` file to the ``users``
```

Comment:
What kinds of files can I use for this? just json? xml? binary? txt? If there's a reference describing this, I would link to it (and if not, you can add that to our missing-links file in ``source/usage-examples//``